es6.md

```diff
@@ -145,3 +145,51 @@ interface ClassExpression <: Class, Expression {
     type: "ClassExpression";
 }
 ```
+
+# Modules
+
+## ImportDeclaration
+
+**FIXME:** Esprima's behavior differs from Acorn's and SpiderMonkey's behavior in that `source` is of type `ModuleSpecifier`, but is otherwise a `Literal`.
+
+```js
+interface ImportDeclaration <: Declaration {
+    specifiers: [ ImportSpecifier ];
+    source: Literal; // Or ModuleSpecifier? (Esprima deviation)
+}
+```
+
+An import declaration, e.g., `import foo from "mod";`.
+
+## ImportSpecifier
+
+```js
+interface ImportSpecifier {
+    imported: Identifier;
+    local: Identifier;
+}
+```
+
+An imported variable binding, e.g., `{foo}` in `import {foo} from "mod"` or `{foo as bar}` in `import {foo as bar} from "mod"`. The `imported` field represents the imported name from the module. The `local` field represents the bindings imported into the module's global scope. If it is a basic named import, such as in `import {foo} from "mod"`, both `imported` and `local` are equivalent `Identifier` nodes; in this case an `Identifier` node representing `foo`. If it is an aliased import, such as in `import {foo as bar} from "mod"`, the `imported` field is an `Identifier` node representing `foo`, and the `local` field is an `Identifier` node representing `bar`.
```

Comment:
> The `imported` field represents the imported name from the module.

Reads slightly confusing. Perhaps: "The `imported` field refers to the name of the export coming from the source module."

> The `local` field represents the bindings imported into the module's global scope

Let's say "the local module scope" rather than "global scope"